src/Nethermind/Nethermind.HealthChecks/ClHealthRequestsTracker.cs

```diff
@@ -0,0 +1,74 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Nethermind.Api;
+using Nethermind.Core;
+using Nethermind.Logging;
+
+namespace Nethermind.HealthChecks;
+
+public class ClHealthRequestsTracker(ITimestamper timestamper, int maxIntervalClRequestTime, ILogger logger)
+    : IEngineRequestsTracker, IClHealthTracker
+{
+    private const int ClUnavailableReportMessageDelay = 5;
+
+    private DateTime _latestForkchoiceUpdated = timestamper.UtcNow;
+    private DateTime _latestNewPayload = timestamper.UtcNow;
+
+    private Timer _timer;
+
+    public Task StartAsync()
+    {
+        _timer = new Timer(ReportClStatus, null, TimeSpan.Zero,
+            TimeSpan.FromSeconds(ClUnavailableReportMessageDelay));
+
+        return Task.CompletedTask;
+    }
+
+    private Task StopAsync()
+    {
+        _timer.Change(Timeout.Infinite, 0);
+
+        return Task.CompletedTask;
+    }
+
+    public async ValueTask DisposeAsync()
+    {
+        await StopAsync();
+        await _timer.DisposeAsync();
+    }
```

Comment:
I think we can inline `StopAsync` and make it sync:

```suggestion
    public async ValueTask DisposeAsync()
    {
        _timer.Change(Timeout.Infinite, 0);
        await _timer.DisposeAsync();
    }
```

Even then, what is the point of `.Change` if it's going to be immediately disposed? I might be ignoring some `Timer` best practices though.

Reply:
As I remember it will still work for some time. Anyways, let's keep it like it was before